libs/sdk-ui-dashboard/styles/scss/scheduled_mail.scss

```diff
@@ -306,3 +382,199 @@ $vertical-space: 10px;
         }
     }
 }
+
+.gd-schedule-email-dialog-attachments {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: 10px;
+
+    & > div:not(:last-child) {
+        margin-right: 20px;
+    }
+}
+
+.gd-schedule-dropdown {
+    &-button {
+        display: flex;
+        align-items: center;
+        height: 30px;
+        margin: 0;
+        padding: 0;
+        border: none;
+        outline: inherit;
+        font: inherit;
+        color: var(--gd-palette-complementary-7, #687581);
+        background: none;
+        text-decoration: underline;
+        font-size: 14px;
+        cursor: pointer;
+
+        &-icon {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            align-items: center;
+            height: 100%;
+            margin-right: 6px;
+            flex-shrink: 0;
+        }
+    }
+
+    &-body {
+        display: flex;
+        flex-direction: column;
+        height: inherit;
+        font-size: 12px;
+
+        @media #{$small-only} {
+            width: 100vw;
+            max-height: calc(100vh - 30px);
+        }
+    }
+
+    &-header {
+        display: flex;
+        flex: 0 0 auto;
+        align-items: center;
+        padding-left: 10px;
+        padding-right: 10px;
+        overflow: hidden;
+        height: 35px;
+        white-space: nowrap;
+        color: var(--gd-palette-complementary-7, #94a1ad);
+        background: var(--gd-palette-complementary-2, #ebeff4);
```

Comment:
I believe that the fallback colors here are already in some of our variables. Please used them if possible. You may also find that there is already some variable with the same complementary and fallback. Use for others styles as well if applicable

e.g. `$is-focused-background`, `$default-gd-color-state-blank` or `$gd-color-state-blank` may be used?

Reply:
I would suggest to try to fix it in the theming and responsiveness story. Anyway we will update the styles once again when we review the UI with Tomas Muchka. I can explicitly mention that in the story.